vllm/worker/worker.py

```diff
@@ -94,6 +95,24 @@ def init_model(self) -> None:
         # Initialize the model.
         set_random_seed(self.model_config.seed)
 
+        self.model = get_model(self.model_config, self.device_config,
```

Comment:
This block of code is in https://github.com/vllm-project/vllm/blob/main/vllm/worker/model_runner.py#L84, I don't think it should be here.